### PR TITLE
Main cleanup

### PR DESCRIPTION
**Pull request by goodnewsj62**
moved ANSI functions in main to it's own mod. 
[issue](https://github.com/watthedoodle/hyperfetch/issues/2)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -9,6 +8,7 @@ mod os;
 mod shell;
 mod title;
 mod uptime;
+mod stdout_utils;
```
Comment (watthedoodle) [inline src/main.rs]: please rename this module to `ansi`

**Review by watthedoodle — CHANGES_REQUESTED** [review]
Good work @goodnewsj62 :+1: I only request a name change.

I think this PR has merge conflicts, but I can sort that out.
Once the requested change is done I'll merge the PR :+1:

File: src/main.rs (hunk 1 above)
Comment (goodnewsj62) [reply src/main.rs]: okay i will do so

**Review by watthedoodle — APPROVED** [review]
Excellent stuff! thanks :+1:

**Pull request merged**